flow/cmd/api.go

```diff
@@ -246,18 +249,56 @@ func APIMain(ctx context.Context, args *APIServerParams) error {
 		return fmt.Errorf("unable to kill existing scheduler flows: %w", err)
 	}
 
-	workflowID := fmt.Sprintf("scheduler-%s", uuid.New())
-	workflowOptions := client.StartWorkflowOptions{
-		ID:        workflowID,
-		TaskQueue: taskQueue,
-	}
+	if args.RunScheduledTasksWithoutTemporal {
+		go func() {
```

Comment:
should probably move this logic into `internal` but wanted to get something out that can be reviewed before tidying up on the level of which files code belongs in